dind/start.sh

```diff
@@ -95,8 +95,45 @@ start_master ()
   docker tag k8s.gcr.io/kube-proxy:$(cat /docker_version) k8s.gcr.io/kube-proxy-amd64:$(cat /docker_version)
   docker tag k8s.gcr.io/kube-scheduler:$(cat /docker_version) k8s.gcr.io/kube-scheduler-amd64:$(cat /docker_version)
 
+  cat <<EOPOLICY > /etc/kubernetes/audit-policy.yaml
+apiVersion: audit.k8s.io/v1beta1
+kind: Policy
+omitStages:
+  - "RequestReceived"
+rules:
+- level: RequestResponse
+  resources:
+  - group: "" # core
+    resources: ["pods", "secrets"]
+  - group: "extensions"
+    resources: ["deployments"]
+EOPOLICY
+  chmod 0600 /etc/kubernetes/audit-policy.yaml
+  cat <<EOF > /etc/kubernetes/kubeadm.conf
+#apiServerExtraArgs:
+#  cert-extra-sans: $1
```

Comment:
any reason for these commented lines?

Reply:
I'm still working on passing cert-extra-sans... the fileformat and command line args have changed... still hunting down the v1alpha3 format.

Reply:
This is fixed now, it had to be a list... and 'kubernetes' also needed to be added